OpenSim/Common/MapObject.h

```diff
@@ -0,0 +1,97 @@
+/* -------------------------------------------------------------------------- *
+ *                          OpenSim: MapObject.h                              *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2019 Stanford University and the Authors                *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#ifndef OPENSIM_MAP_OBJECT_H_
+#define OPENSIM_MAP_OBJECT_H_
+ // INCLUDES
+#include "Object.h"
+
+namespace OpenSim {
+/**
+* A class for representing an item in a Map
```

Comment:
It is not clear to me why this is a useful abstraction for OpenSim users or simplifies the implementation for the dev team. In fact, I think it may make things more confusing. For example, Object already has a name attribute, so now the `MapItem` has `name`, `from_name` and `to_value` all as string attributes that are not very informative and are going to be easily confused. Furthermore, `MapItem` itself is a dead-end if  we need to associate more information like offset Transform. In that case, we would have to then create another object to be the value, which would again have a name, etc... with another level of indirection. I just don't see the benefits of having `MapObject`. 

Instead, I would advocate replacing `MapObject` with a data reader specific setup file that has [two lists](https://github.com/opensim-org/opensim-core/blob/a99616a6128c8d4300f589db9effb2ac3f9f3b38/OpenSim/Common/Test/testIMUHelper.cpp#L40-L41)  as properties (lists) that allows sensor data to be renamed/labeled as part of the reading process. From that point on, OpenSim will expect the experimental and model frames to have the same names (like we expect for markers, forces, coordinates, ...). That seems the most straightforward.